### ISSUE TITLE
SDL headers?

### ISSUE THREAD

**Issue by aurelVZ** [Cheeseness/hidden-star#8]
Hi
How to compile this game on Windows i cannot found SDL h

**Comment by Cheeseness** [comment]
Hi! SDL is a third party library. You can find it at libsdl.org

[SDL_Image](https://www.libsdl.org/projects/SDL_image/) and [SDL_Mixer](https://www.libsdl.org/projects/SDL_image/), which are separate projects.

If you haven't spotted it, the game has a cmake build system. I haven't written compilation instructions, but if you get stuck, I can point you in the right direction.

Although I'm not actively working on Hidden Star right now, the primary repo for this project is [over here](https://gitlab.com/Cheeseness/hidden-star)

**Comment by aurelVZ** [comment]
so ineed those dlls to work ?

2021-06-16 3:34 GMT+02:00, Cheeseness ***@***.***>:

**Comment by Cheeseness** [comment]
> so ineed those dlls to work ?
Yes, without them, the game won't compile. If you're not up for compiling the game, there are builds [here](https://cheeseness.itch.io/hidden-star)
